source/guides/stacks/dbrs.rst

```diff
@@ -0,0 +1,599 @@
+.. _dbrs:
+
+
+Database Reference Stack
+########################
+
+This guide covers the hardware and installation requirements for using the :abbr:`DBRS (Database Reference Stack)`, and provides getting started configuration examples. The steps in this guide use |CL-ATTR| as the host system.
+
+
+
+.. contents::
+   :local:
+   :depth: 1
+
+Overview
+********
+
+The Database Reference Stack is integrated, highly-performant, open source, and optimized for 2nd generation Intel® Xeon® Scalable Processors and Intel® Optane™ DC Persistent Memory. This open source community release is part of an effort to ensure developers have easy access to the features and functionality of Intel Platforms.
+
+Stack Features
+==============
+
+Current supported  database applications are Cassandra* and Redis*.
+
+DBRS with Cassandra can be deployed as a standalone container or inside a Kubernetes* cluster.
+
+The Redis stack application is enabled for a multinode Kubernetes environment, using AEP persistent memory DIMM in fsdax mode for storage.
+
+
+
+Hardware Requirements
+*********************
+
+* Intel Xeon Scalable Platform with Intel C620 chipset series
+* 2nd Gen Intel Xeon Scalable processor CPU (Intel Optane DC PMM-enabled stepping) Provides cache & memory control.  Intel Optane DC persistent memory works only on systems powered by 2nd Generation Intel Xeon Platinum or Gold processors.
+* BIOS with Reference Code
+* Intel Optane DC persistent memory
+
+Hardware configuration used in stacks development
+=================================================
+
+* Intel Server System R2208WFTZSR
+* BIOS with Reference Code
+  * BIOS ID: SE5C620.86B.0D.01.0438.032620191658
+  * BMC Firmware: 1.94.6b42b91d
+  * Apache Pass Firmware: 1.2.0.5310
+* 2x Intel Xeon Platinum 8268 Processor
+* Intel SSD DC S5600 Series 960GB 2.5in SATA Drive
+* 64 GB RAM - Distributed in 4x 16 GB DDR4 DIMM's
+* 2x Intel Optane DC Persistent Memory 256GB Module
+* 1-1-1 Layout 8 AEP : 1 RAM ratio
+
+
+.. list-table:: **Table 1. IMC**
+   :widths: 16,16,16,16,16,16
+   :header-rows: 1
+
+   * - Channel 2
+     - Channel 2
+     - Channel 1
+     - Channel 1
+     - Channel 0
+     - Channel 0
+
+    * - Slot 1
```

Comment:
Move back one space all lines from 65 to 77. It's Python. White space matters.  😉

Reply:
Done. Removed 12 whitespaces.